packages/@aws-cdk/aws-ecs-patterns/lib/base/queue-processing-service-base.ts

```diff
@@ -147,6 +147,24 @@ export interface QueueProcessingServiceBaseProps {
    * @default - Automatically generated name.
    */
   readonly family?: string;
+
+  /**
+   * The maximum number of tasks, specified as a percentage of the Amazon ECS
+   * service's DesiredCount value, that can run in a service during a
+   * deployment.
+   *
+   * @default - 200
```

Comment:
We should actually set these default values in the constructor down below for `QueueProcessingServiceBase`, right?

Reply:
Yes, we can do that. In application-load-balanced-service it is not set explicitly: [code](https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/aws-ecs-patterns/lib/ecs/application-load-balanced-ecs-service.ts#L129-L130) and the default from Ec2Service and FargateService is assumed. Should we leave it so it is consistent ?

Reply:
For `FargateService`, looks like `maxHealthPercent` and `minHealthPercent` are not used at all https://github.com/aws/aws-cdk/blob/456c469dd4b92a6a863e4c40125adf573d4df239/packages/%40aws-cdk/aws-ecs/lib/fargate/fargate-service.ts#L130.

In `Ec2Service`, we do use these values: https://github.com/aws/aws-cdk/blob/456c469dd4b92a6a863e4c40125adf573d4df239/packages/%40aws-cdk/aws-ecs/lib/ec2/ec2-service.ts#L158

Maybe make it explicit that these will follow the default values in `Ec2Service` if not provided

Reply:
In FargateService they are passed implicitly [here](https://github.com/aws/aws-cdk/blob/456c469dd4b92a6a863e4c40125adf573d4df239/packages/%40aws-cdk/aws-ecs/lib/fargate/fargate-service.ts#L147) and then used [here](https://github.com/aws/aws-cdk/blob/456c469dd4b92a6a863e4c40125adf573d4df239/packages/%40aws-cdk/aws-ecs/lib/base/base-service.ts#L343). 

I think they are used in Ec2Service explicitly to handle the [daemon case](https://github.com/aws/aws-cdk/blob/456c469dd4b92a6a863e4c40125adf573d4df239/packages/%40aws-cdk/aws-ecs/lib/ec2/ec2-service.ts#L190-L191).

I can either set it explicitly when calling the constructor or I can document it that it is using whatever the default of the underlying Ec2/Fargate construct is.

Reply:
Thanks! Totally missed the `...props` part. Based on https://docs.aws.amazon.com/AmazonECS/latest/developerguide/service_definition_parameters.html, I think those 2 configurations are indeed not used at all even if you pass them in. It just happens to be the case that both `Ec2Service` and `FargateService ` extends `BaseService`. But overall I agree with documenting the defaults are provided by the underlying Ec2/Fargate construct.

Thanks for the PR!